pages/establishment/licence-fees/personal/views/row.jsx

```diff
@@ -4,16 +4,23 @@ import fetch from 'r2';
 import { Inset, Snippet, Form } from '@asl/components';
 import { Button } from '@ukhomeoffice/react-components';
 import format from 'date-fns/format';
+import Markdown from 'react-markdown';
 import { dateFormat } from '../../../../../constants';
 
-function UpdatingForm({ id, billable, onCancelClick, formFields }) {
+const schema = {
+  comment: {
+    inputType: 'textarea'
+  }
+};
+
+function UpdatingForm({ id, waived, onCancelClick, formFields }) {
   return (
     <Fragment>
       {
         formFields
       }
       <input type="hidden" name="pilId" value={id} />
-      <input type="hidden" name="billable" value={billable} />
+      <input type="hidden" name="waived" value={!waived} />
```

Comment:
This feels a bit weird but maybe I'm just misunderstanding it? We submit a value of waived but it's flipped from the value that was loaded?

Reply:
Yes, because the form is to turn it on and off, so it needs to flip somewhere.

Reply:
Ah ok, didn't realise this was the toggle.